src/Servers/Kestrel/Core/src/Internal/Http2/Http2Connection.Generated.cs

```diff
@@ -21,5 +21,6 @@ internal partial class Http2Connection
         private static ReadOnlySpan<byte> TeBytes => new byte[2] { (byte)'t', (byte)'e' };
         private static ReadOnlySpan<byte> TrailersBytes => new byte[8] { (byte)'t', (byte)'r', (byte)'a', (byte)'i', (byte)'l', (byte)'e', (byte)'r', (byte)'s' };
         private static ReadOnlySpan<byte> ConnectBytes => new byte[7] { (byte)'C', (byte)'O', (byte)'N', (byte)'N', (byte)'E', (byte)'C', (byte)'T' };
+        private static ReadOnlySpan<byte> ProtocolBytes => new byte[9] { (byte)':', (byte)'p', (byte)'r', (byte)'o', (byte)'t', (byte)'o', (byte)'c', (byte)'o', (byte)'l' };
```

Comment:
Why not do `private static ReadOnlySpan<byte> ProtocolBytes => ":protocol"u8;` instead of manually defining this? For context this is what we do in the HTTP/3 stuff.

Reply:
The u8 syntax is brand new and this generated code hasn't been updated. That's a style change we'd do separately.